epistemix_platform/tests/use_cases/test_upload_results.py

```
@@ -0,0 +1,680 @@
+        # Verify no upload was attempted
+        mock_results_repository.upload_results.assert_not_called()
+        mock_run_repository.save.assert_not_called()
+
+    def test_upload_fails_when_results_directory_does_not_exist(
+        self,
+        mock_run_repository,
+        mock_job_repository,
+        mock_results_packager,
+        mock_results_repository,
+        mock_time_provider,
+        sample_job,
```

Ruff (0.14.1)

Unused method argument: `sample_job`

(ARG002)